com.unity.shadergraph/Editor/Drawing/SearchWindowAdapter.cs

```diff
@@ -15,6 +17,52 @@ public SearchWindowAdapter(string title) : base(title)
         {
             m_DefaultItemTemplate = Resources.Load<VisualTreeAsset>("SearcherItem");
         }
+
+        public override SearcherItem OnSearchResultsFilter(IEnumerable<SearcherItem> searchResults, string searchQuery)
+        {
+            // Sort results by length so that shorter length results are prioritized
+            // prevents entries with short names getting stuck at end of list after entries with longer names when both contain the same word
+            searchResults = searchResults.OrderBy(x => x.Name.Length).ToList();
```

Comment:
Is this intended to address https://fogbugz.unity3d.com/f/cases/1348390/ as well?  Seems like it might.

Reply:
Oh right good call will add that to the description, thanks Jesse!

Reply:
Added now 👍